pp-2023/src/app/kmeans/exportButton.js

```diff
@@ -0,0 +1,52 @@
+import XLSX from "xlsx";
+
+// Diese Funktion implementiert die Logik zum parsen der Ergebnis Daten in ein Excel Sheet.
+// Außerdem wird hier der Button zum Download implementiert
+
+export const ExportExcelFile = ({result}) => {
+
+const handleOnButtonClick = () => {
+
+    //Abfrage ob ein Ergebnis aus lokaler Berechnung vorliegt
+if (result.groups){
+    let k = result.options.k; // Abfrage wieviele Cluster das Ergebnis enthält
```

Comment:
bitte die Variablen etwas lesbarere gestalten, tu ich mich auch schwer mit

Reply:
Variable ist in neustem Commit umbenannt (: